index.html

```diff
@@ -137,12 +138,14 @@ <h3 class="project-title">Multi-Post Stories Gain+Glory</h3>
     
     <section id="about">
       <div id="background-about">
+        <img src="images/illustration-about me-1-desktop@2x.png" alt="" id="about-img-1" class="desktop">
         <div id="about-me"> 
-          <h2 class="titlesClass container" >About me</h2>
-          <p class="container"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+          <h2 class="titlesClass" id="about-me-title">About me</h2>
+          <p class="about-me-text"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
             don't hesitate to contact me.</p>
             <button class="button" id="about-me-button">Get my resume</button> 
         </div>
+        <img src="images/illustration-about-me-2-desktop@2x.png" alt="" id="about-img-2" class="desktop">
```

Comment:
- [ ] [ ]Kindly as requested by the previous reviewer consider adding this image as a background image because The images that are added to improve the UI of the application should be added as background images.